shared/teams/common/activity.tsx

```diff
@@ -34,7 +36,16 @@ const Activity = ({level}: {level: Types.ActivityLevel}) =>
     </Kb.Box2>
   )
 
-export const ModalTitle = ({title, teamname, avatarFilepath, avatarCrop}: Props) => {
+export const ModalTitle = ({title, teamID}: Props) => {
+  const teamname =
+    title == 'Enter team info'
```

Comment:
oops left a commit comment re this instead of a PR comment. See the commit comment, but please change this.